hardware_interface/src/resource_manager.cpp

```diff
@@ -762,24 +760,23 @@ class ResourceStorage
 
   // Update rate of the controller manager, and the clock interface of its node
   // Used by async components.
-  unsigned int cm_update_rate_;
+  unsigned int cm_update_rate_ = 100;
```

Comment:
Maybe it is better to initialize it with `nan` and check before using it, if not maybe things might run and it would be hard to find that it is running at a different frequency if this is not set properly upon overriding. What's your opinion on this?

Reply:
This is the same behavior that can happen before. Nevertheless, when we are reading URDF the value will be set from CM. So now in standard setup, Storage is running on 100 Hz, but empty before any HW is loaded, so it doesn't make any difference.

Reply:
I am also in to remove this alltogeather, but it will be a lot of change so if it is OK with you, I would do in follow-up

Reply:
I understand that this wouldn't happen in the normal setup, but it might happen in the simulators where we have to override the `load_and_initialize_components` method and if this is not set then it will be updated in the wrong frequency right? or if we pass an already constructed ResourceManager object?. What do you think?. If you think this is not an issue, it is fine for me

Reply:
Ok then we can do it in the followup!

Reply:
Issues opened!